dags.yaml

```diff
@@ -1343,3 +1343,19 @@ bqetl_glam_export:
   tags:
     - impact/tier_2
   description: DAG to prepare GLAM data for public export.
+
+bqetl_crash:
+  schedule_interval: 0 2 * * *
+  default_args:
+    depends_on_past: false
+    email:
+      - dthorn@mozilla.com
+      - telemetry-alerts@mozilla.com
+    email_on_failure: false
+    email_on_retry: true
+    owner: dthorn@mozilla.com
+    retries: 2
+    retry_delay: 30m
+    start_date: "2023-12-10"
+  tags:
+    - impact/tier_2
```

Comment:
This will be used to generate a top-crash-signatures report, which will be used in conjunction with the existing top crashers report from crash ingestion.